.circleci/config.yml

```diff
@@ -432,12 +432,7 @@ jobs:
       - run:
           name: snapshots
           command: |
-            pnpm snapshots
-            if git diff --exit-code snapshots > /dev/null; then
-              [ -z "$(git ls-files --others --exclude-standard snapshots)" ] || echo "export SNAPSHOTS_STATUS=1" >> "$BASH_ENV"
-            else
-              echo "export SNAPSHOTS_STATUS=1" >> "$BASH_ENV"
```

Comment:
This is change to how we do error handling in this job. Previously we would set error codes for each job & only fail after each job had been completed. Now we will do that for some steps except for this step which will fail immediately.